### ISSUE TITLE
Skeletonize the Repository

### ISSUE THREAD

**Issue by mcdonnnj** [cisagov/pca-core#6]
# 💡 Summary #

This repository should be skeletonized to descend from [cisagov/skeleton-python-library](https://github.com/cisagov/skeleton-python-library). Once that is done the Docker elements should be broken out into their own `cisagov/pca-core-docker` repository which should be started from [cisagov/skeleton-docker](https://github.com/cisagov/skeleton-docker).

## Motivation and context ##

All of our active projects that are able should descend from an appropriate skeleton repository. This will ensure they follow organization development standards and practices as well as allow for automated updates from [cisagov/action-lineage](https://github.com/cisagov/action-lineage).

## Implementation notes ##

Skeletonizing should be done by following [our documentation](https://github.com/cisagov/development-guide/blob/develop/project_setup/skeletonize-existing-repository.md) and getting feedback from dev team members. The new `cisagov/pca-core-docker` repository should be started following our [project setup documentation](https://github.com/cisagov/development-guide/tree/develop/project_setup#using-the-skeleton-tool-to-start-a-new-repository-).

## Acceptance criteria ##

- [ ] Repository has a `skeletonize` branch that properly integrates [cisagov/skeleton-python-library](https://github.com/cisagov/skeleton-python-library) using [or skeletonizing documentation](https://github.com/cisagov/development-guide/blob/develop/project_setup/skeletonize-existing-repository.md)
- [ ] A PR from that branch is approved by members of @cisagov/team-ois and merged into this repository
- [ ] The Dockerfile and any Docker related pieces/documentation are moved out to their own `cisagov/pca-core-docker` repository that is based on [cisagov/skeleton-docker](https://github.com/cisagov/skeleton-docker) as described in our [project setup documentation](https://github.com/cisagov/development-guide/tree/develop/project_setup#using-the-skeleton-tool-to-start-a-new-repository-)

**Comment by mcdonnnj** [comment]
@BenBreaksThings Please assign this as you see fit.
